src/main/java/uk/gov/hmcts/reform/civil/utils/HearingUtils.java

```diff
@@ -54,4 +59,55 @@ public static BigDecimal getFastTrackFee(int claimFee) {
             return new BigDecimal(34600);
         }
     }
+
+    public static String getHearingType(CaseData caseData) {
+        switch (caseData.getChannel()) {
+            case IN_PERSON:
+                return caseData.getHearingLocation().getValue().getLabel();
+            case VIDEO:
+                return "videoconference";
+            case TELEPHONE:
+                return "telephone";
+            default:
+                return "not defined";
+        }
+    }
+
+    public static String getHearingDuration(CaseData caseData) {
```

Comment:
Lots of branching in here due to the switch statement. Please add a dedicated unit test to ensure the output matches expectations for all branches.